src/Sylius/Bundle/TranslationBundle/spec/Doctrine/ODM/MongoDB/TranslatableResourceRepositorySpec.php

```diff
@@ -25,6 +25,8 @@
 /**
  * Doctrine ODM MongoDB driver translatable document repository spec.
  *
+ * @require \Doctrine\ODM\MongoDB\DocumentManager
```

Comment:
There is no MongoDB extension for PHP 7 so we must skip these specifications that are based on it.